### DISCUSSION TITLE
React Popper placement does not work inside of Transition

### DISCUSSION THREAD

**Discussion by Ghirigoro** [tailwindlabs/headlessui#384] [Help]
In the Popover page the basic example wraps the `Popover` in a `Transition`. Later in that page it recommends using `react-popper` for placement, but if you use a `Transition` like in the basic example, `react-popper` breaks and can no longer place the `Popover` (it always gets placed over the top left of the button).

Is `Transition` not supposed to work with `react-popper` or is that a bug? And if it's not a bug should the page be updated to explain that `react-popper` is not supported with transitions (or maybe suggest a supported library or workaround instead)?

**Comment by sebastienbarre** [comment]
Can reproduce that as well, and would love to get an answer.
Thanks

**Reply by sebastienbarre** [reply]
I asked on the Discord server. Maybe somebody will hint at a solution.
https://discordapp.com/channels/674983035946663957/682241064115109992/847473030724845598

**Comment by ycs77** [comment]
Hi! I create a package [Headless UI Float](https://github.com/ycs77/headlessui-float), that can be easy to position floating Headless UI elements using Floating UI (is new version Popper.js), support Transition & Portal.

If you need to float Headless UI element can try it out 😊
